### ISSUE TITLE
Ghostbuster (2016) mismatch to the old movie (1984)

### ISSUE THREAD

**Issue by Haldir31** [Radarr/Radarr#4866]
**Describe the bug**
During the import for existing movie, I have ghostbuster movie with the old version from 1984 and the new one from 2016. Folder name is formatted as "Movie Name (Year)"

**To Reproduce**
1. Create two folder named Ghostbusters (1984) & Ghostbusters (2016)
2. Go in https://<servername>/add/import
3. Start import
4. Both movie are displayed with the expected folder name but the movie match is only for the 1984 version

**Expected behavior**
Based on the movie name and the year, both version should match with their respective year. To have a expected match, folder have to include the IMDB ID. TMDB ID is not able to override the match and use the 2016 version.
 
**Screenshots**
![Capture d’écran 2020-08-13 à 20 42 49](https://user-images.githubusercontent.com/53797644/90174408-3a42fe00-dda6-11ea-9965-64730da9daef.png)

**Platform Information (please complete the following information):**
 - OS: MacOS 10.15.6
 - Mono Version: .Net Core 6.10.0 (Stable Channel)
 - Browser and Version [e.g. chrome, safari] (Only needed for UI issues)
 - Radarr Version: 3.0.0.3406

**Debug Logs**
2020-8-13 20:48:10.7|Debug|Api|[GET] /api/v3/health: 200.OK (0 ms)
2020-8-13 20:48:10.7|Debug|Api|[PUT] /api/v3/config/host: 202.Accepted (27 ms)
2020-8-13 20:48:10.8|Debug|Api|[GET] /api/v3/health: 200.OK (0 ms)
2020-8-13 20:48:16.9|Debug|RootFolderService|Generating list of unmapped folders
2020-8-13 20:48:17.0|Debug|RootFolderService|2 unmapped folders detected.
2020-8-13 20:48:17.0|Debug|Api|[GET] /api/v3/rootFolder: 200.OK (4 ms)
2020-8-13 20:48:19.0|Debug|RootFolderService|Generating list of unmapped folders
2020-8-13 20:48:19.1|Debug|RootFolderService|2 unmapped folders detected.
2020-8-13 20:48:19.1|Debug|Api|[GET] /api/v3/rootFolder: 200.OK (4 ms)
2020-8-13 20:48:25.6|Debug|RootFolderService|Generating list of unmapped folders
2020-8-13 20:48:25.6|Debug|RootFolderService|2 unmapped folders detected.
2020-8-13 20:48:25.6|Debug|Api|[GET] /api/v3/rootFolder/1?timeout=false: 200.OK (4 ms)
2020-8-13 20:48:25.6|Debug|Parser|Parsing string 'Ghostbusters (1984)'
2020-8-13 20:48:25.6|Debug|Parser|Movie Parsed. Ghostbusters - 1984 
2020-8-13 20:48:25.6|Debug|Parser|Release Group parsed: 
2020-8-13 20:48:25.6|Debug|Parser|Languages parsed: Unknown
2020-8-13 20:48:25.6|Debug|QualityParser|Trying to parse quality for Ghostbusters (1984)
2020-8-13 20:48:25.6|Debug|Parser|Quality parsed: Unknown v1
2020-8-13 20:48:25.6|Debug|Parser|Edition parsed: 
2020-8-13 20:48:25.7|Debug|Api|[GET] /api/v3/movie/lookup?term=Ghostbusters%20(1984): 200.OK (56 ms)
2020-8-13 20:48:25.7|Debug|Parser|Parsing string 'Ghostbusters (2016) [tmdb-43074]'
2020-8-13 20:48:25.7|Debug|Parser|Movie Parsed. Ghostbusters - 2016 
2020-8-13 20:48:25.7|Debug|Parser|Release Group parsed: 
2020-8-13 20:48:25.7|Debug|Parser|Languages parsed: Unknown
2020-8-13 20:48:25.7|Debug|QualityParser|Trying to parse quality for Ghostbusters (2016) [tmdb-43074]
2020-8-13 20:48:25.7|Debug|Parser|Quality parsed: Unknown v1
2020-8-13 20:48:25.7|Debug|Parser|Edition parsed: 
2020-8-13 20:48:25.7|Debug|Api|[GET] /api/v3/movie/lookup?term=Ghostbusters%20(2016)%20%5Btmdb-43074%5D: 200.OK (59 ms)
2020-8-13 20:48:31.2|Debug|Api|[GET] /api/v3/health: 200.OK (0 ms)
2020-8-13 20:48:31.2|Debug|Api|[GET] /api/v3/system/status: 200.OK (0 ms)
2020-8-13 20:48:31.2|Debug|Api|[GET] /api/v3/diskspace: 200.OK (6 ms)
2020-8-13 20:48:33.4|Debug|Api|[GET] /api/v3/log/file: 200.OK (6 ms)

**Comment by austinwbest** [comment]
![Annotation 2020-08-19 130508](https://user-images.githubusercontent.com/8321115/90667387-b2814780-e21c-11ea-83d1-9835f402bdca.png)

I added both ghostbusters to a root folder and went to import, it detected them both without issue.

**Comment by bakerboy448** [comment]
Would this help / fix this?

https://github.com/Radarr/Radarr/pull/6870

**Comment by howardjones** [comment]
Yes, this is the same issue. And I would still suggest that "the API gives this order so it not our problem" is not helpful to the user's experience.

**Comment by austinwbest** [comment]
> Yes, this is the same issue. And I would still suggest that "the API gives this order so it not our problem" is not helpful to the user's experience.

Where exactly does it say this is not our problem?

Q clearly said we should change things, as in radarr, as in this is something we can possibly take care of, as in the complete opposite of what you are quoting.

Maybe I am missing some posts on the thread though so please quote the post that says that so I have some clarification if you don't mind bud.

**Comment by howardjones** [comment]
Sure, "On Hold: Metadata API blocking"? Or this: https://www.reddit.com/r/radarr/comments/rnlp69/comment/hpt2u1a

**Comment by bakerboy448** [comment]
Metadata meaning it'd be something to correct in our metadata server if TMDB won't cooperate.

But there's also already a PR going relating to this.

If it's something that would not be fixed - it would be closed.
